qiling/debugger/disassember.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# 
+# Cross Platform and Multi Architecture Advanced Binary Emulation Framework
+# Built on top of Unicorn emulator (www.unicorn-engine.org) 
+from elftools.elf.elffile import ELFFile
+from qiling import *
+from qiling.const import *
+from capstone import *
+
+
+class QlDisassember():
+    def __init__(self, ql:Qiling):
+        pass
+
+    def disasm_all_lines(self, ql:Qiling, seg_name='.text'):
```

Comment:
disam_elf , disasm_pe

this part is rather file format specific and not OS specific ?